pyart/retrieve/__init__.py

```diff
@@ -27,4 +27,10 @@
 from .gate_id import map_profile_to_gates, fetch_radar_time_profile
 from .simple_moment_calculations import calculate_snr_from_reflectivity
 
+try:
```

Comment:
Why does this need to be optionally imported?  Everything in the module is a required dependency